data/data/agent/files/usr/local/share/assisted-service/images.env.template

```diff
@@ -1,3 +1,3 @@
+OS_IMAGES=[{"openshift_version":"4.10","cpu_architecture":"x86_64","url":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-rc.1/rhcos-4.10.0-rc.1-x86_64-live.x86_64.iso","rootfs_url":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.10.0-rc.1/rhcos-4.10.0-rc.1-x86_64-live-rootfs.x86_64.img","version":"410.84.202201251210-0"},{"openshift_version":"4.11","cpu_architecture":"x86_64","url":"https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos-4.11/411.85.202203181601-0/x86_64/rhcos-411.85.202203181601-0-live.x86_64.iso","rootfs_url":"https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos-4.11/411.85.202203181601-0/x86_64/rhcos-411.85.202203181601-0-live-rootfs.x86_64.img","version":"411.85.202203181601-0"},{"openshift_version":"4.12","cpu_architecture":"x86_64","url":"https://releases-rhcos-art.cloud.privileged.psi.redhat.com/storage/releases/rhcos-4.12/412.86.202208170136-0/x86_64/rhcos-412.86.202208170136-0-live.x86_64.iso","rootfs_url":"https://releases-rhcos-art.cloud.privileged.psi.redhat.com/storage/releases/rhcos-4.12/412.86.202208170136-0/x86_64/rhcos-412.86.202208170136-0-live-rootfs.x86_64.img","version":"412.86.202208170136-0"},{"openshift_version":"4.12","cpu_architecture":"aarch64","url":"https://releases-rhcos-art.apps.ocp-virt.prod.psi.redhat.com/storage/releases/rhcos-4.12-aarch64/412.86.202209061132-0/aarch64/rhcos-412.86.202209061132-0-live.aarch64.iso","rootfs_url":"https://releases-rhcos-art.apps.ocp-virt.prod.psi.redhat.com/storage/releases/rhcos-4.12-aarch64/412.86.202209061132-0/aarch64/rhcos-412.86.202209061132-0-live-rootfs.aarch64.img","version":"412.86.202209061132-0"}]
```

Comment:
Just as a heads-up, we are going to stop hard-coding this and generate it on the fly in https://github.com/openshift/installer/pull/6441 - but the current implementation also hard-codes the architecture in the code. So you might want to rebase on that and add CPU Arch awareness there.